py/selenium/webdriver/firefox/webdriver.py

```diff
@@ -247,7 +246,11 @@ def install_addon(self, path, temporary=None) -> str:
 
                 driver.install_addon('/path/to/firebug.xpi')
         """
-        payload = {"path": path}
+        file_ = open(path, 'rb')
```

Comment:
This would be better as a context manager

```python
with open(path, rb) as file:
     # the encoding part
```

Context managers handle the opening and closing for you in a much better way and will be GC'ed better